index.js

```diff
@@ -49,5 +49,6 @@ module.exports = {
 			groups: ['builtin', 'external', 'parent', 'sibling', 'index'],
 			alphabetize: { order: 'asc' },
 		}],
+		'no-shadow': ['off'],
```

Comment:
I think this rule should only be turned off in the typescript lint rules - if for some reason this lint config is used for a javascript project the no-shadow rule will be completely disabled.

Reply:
Somehow it really wants it: 
![image](https://user-images.githubusercontent.com/4455751/152803442-fa2235d3-75bc-4c16-a80d-be46a69de772.png)

Reply:
Or did you mean moving it to the typescript.js file only?